schedule/publiccloud/mau-extratests-docker-publiccloud.yml

```diff
@@ -0,0 +1,24 @@
+---
+name: mau-extratests-docker-publiccloud
+description: |
+  Run mau-extratests-docker on public cloud instances
+schedule:
+  - boot/boot_to_desktop
+  - publiccloud/download_repos
+  - publiccloud/ssh_interactive_init
+  - publiccloud/register_system
+  - publiccloud/transfer_repos
+  - publiccloud/patch_and_reboot
+  - publiccloud/img_proof
+  - publiccloud/ssh_interactive_start
+  - publiccloud/instance_overview
+  - console/system_prepare
+  - console/prepare_test_data
+  - console/consoletest_setup
+  - containers/podman
+  - containers/docker
+  - containers/docker_runc
```

Comment:
We should talk about `docker_compose` also. cc: #11502

Reply:
not sure about that. what s the problem with that module? i see a lot of (fails)[https://openqa.suse.de/tests/overview?arch=&machine=&modules=docker_compose&groupid=139&groupid=142&groupid=163&groupid=164&groupid=157&groupid=155&groupid=240&groupid=167&groupid=169&groupid=170&groupid=190&groupid=248&groupid=245&groupid=246&groupid=247&groupid=210&groupid=272&groupid=273&groupid=162&groupid=152&groupid=196&groupid=183&groupid=110&groupid=129&groupid=116&groupid=257&groupid=130&groupid=262&groupid=268&groupid=263&groupid=264&groupid=219&groupid=274&groupid=275&groupid=276&groupid=131&groupid=259&groupid=118&groupid=143&groupid=111&groupid=222&groupid=113&groupid=156&groupid=146&groupid=114&groupid=112&groupid=115&groupid=213&groupid=221&groupid=249&groupid=159&groupid=166&groupid=233&groupid=231&groupid=177&groupid=236&groupid=176&groupid=237&groupid=161&groupid=1126045&groupid=281&groupid=214&groupid=107&groupid=100&groupid=99&groupid=282&groupid=215&groupid=106&groupid=53&groupid=41&groupid=193&groupid=191&groupid=283&groupid=216&groupid=150&groupid=149&groupid=284&groupid=217&groupid=141&groupid=140&groupid=144&groupid=187&groupid=232&groupid=165&groupid=34&groupid=54&groupid=108&groupid=218&groupid=280#]

Reply:
i am resolving this for now as it can be part of another PR later